### PR TITLE
PP-7570 Clean up logging

### PR DESCRIPTION
**Pull request by stephencdaly**
As we now attach the correlation id, user, service, and account info to all log lines when it is available using AsyncLocalStorage, we no need to attach this information to individual log lines.

See b18c7571d774ab3c591d4bf614f24ac38a923de0 for implementation of this.

Remove occurrences where we log this info in individual log lines.

### CODE REVIEW

File: app/services/auth.service.js [hunk 1]
```diff
@@ -104,20 +103,20 @@ function setSessionVersion (req) {
...
-  if (!isValid) logger.info(`[${req.correlationId}] Invalid session version for user. User session_version: ${lodash.get(req, 'user.sessionVersion', 0)}, session version ${lodash.get(req, 'session.version')}`)
+  if (!isValid) logger.info(`Invalid session version for user. User session_version: ${lodash.get(req, 'user.sessionVersion', 0)}, session version ${lodash.get(req, 'session.version')}`)
   return isValid
 }
 
 function addUserFieldsToLogContext (req, res, next) {
   if (req.user) {
     addLoggingField(USER_EXTERNAL_ID, req.user.externalId)
-    addLoggingField('is_internal_user', req.user.internalUser)
+    addLoggingField('internal_user', req.user.internalUser)
```
Comment (kbottla) [inline app/services/auth.service.js:119]: This affects few selfservice Splunk dashboards

**Review by kbottla — APPROVED** [review]
Looks good to me. Renaming `is_internal_user` log field to `internal_user` affects few dashboards. Happy to rename here and update dashboards or change it back to `is_internal_user`

File: app/services/auth.service.js (hunk 1 above)
Comment (stephencdaly) [reply app/services/auth.service.js:119]: we inconsistently have `is_internal_user` and `internal_user` for different log lines, so I picked the one we used for the transaction search log lines. Good point though, I'll check if any dashboards need updating.

Comment (stephencdaly) [reply app/services/auth.service.js:119]: I'll keep it as `internal_user` and update the dashboards

**Pull request merged**